### ISSUE TITLE
How to make one-step ahead predictions for autoregressive models?

### ISSUE THREAD

**Issue by mtomic123** [jdb78/pytorch-forecasting#1183]
I trained a DeepVAR model with 10 features. I am looking to make n one-step ahead predictions in an autoregressive manner. This can be both on validation set or completely new data.

I found the `decode_autoregressive()` method in the `AutoRegressiveBaseModel` class, however it is unclear to me how to use this method as it requires a bunch of inputs. Is there a more straightforward way to make one-step ahead predictions iteratively?
